desktop/src/test/java/bisq/desktop/main/market/trades/TradesChartsViewModelTest.java

```diff
@@ -121,10 +107,11 @@
             null,
             1
     );
-
     @Before
     public void setup() throws IOException {
-
+        tsm = mock(TradeStatisticsManager.class);
```

Comment:
Please avoid acronyms.

Reply:
yeah that's what it was called before, will change.